docker-compose.yaml

```
@@ -3,32 +3,32 @@
 version: "3"
```

I believe we can also remove the `version` attribute since it's deprecated.

```suggestion
version: "3"
```

---

Excellent